doc/api/http.markdown

```diff
@@ -64,10 +64,10 @@ a `'close'` event or a special `'agentRemove'` event. This means that if
 you intend to keep one HTTP request open for a long time and don't
 want it to stay in the pool you can do something along the lines of:
 
-    http.get(options, function(res) {
+    http.get(options, (res)=> {
```

Comment:
In this case it's probably fine to omit the parens, as well — `http.get(options, res => {`